src/System.Private.Uri/src/System/Uri.Mono.cs

```diff
@@ -0,0 +1,7 @@
+namespace System
+{
+    public partial class Uri
+    {
+        private static readonly bool IsWindowsSystem = PlatformHelper.IsWindows;
+    }
+}
```

Comment:
You cannot compile both Uri.Unix.cs and Uri.Windows.cs at the same time, as the definition of `IsWindowsSystem` would conflict, and the idea of the `mono` configuration is that the same assembly works both for windows and unix.